client/src/utils/messaging.js

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+import firebase from "firebase/app";
+import "firebase/messaging";
+
+class MessagingHandler {
+  constructor() {
+    // Information available in the firebase console
+    this.app = firebase.initializeApp({
```

Comment:
Is this possible to DRY with https://github.com/ValFraNath/guacamole/pull/189/files#diff-c49bbee4ec41262d0cf13f76eeb02b9496e129ec768738fcff52402892f89361R17?

Reply:
Good question, I will try